README.md

```diff
@@ -266,3 +274,24 @@ service will be tracked.
 To disable tracking, simply remove ``cf_deployment_tracker.track()`` and ``metrics_tracker_client.track()`` from the
 ``run.py`` file in the top level directory.
 
+# Links
+* [Demo on Youtube](https://youtu.be/b-94B3O1czU)
+* [Watson Conversation](https://www.ibm.com/watson/services/conversation/)
+* [Watson Discovery](https://www.ibm.com/watson/services/discovery/)* [IBM Cloudant db](https://www.ibm.com/cloud/cloudant)
+* [Slack](https://slack.com)
+
+# Learn more
+
+* **Artificial Intelligence Code Patterns**: Enjoyed this Code Pattern? Check out our other [AI Code Patterns](https://developer.ibm.com/code/technologies/artificial-intelligence/).
+* **Data Analytics Code Patterns**: Enjoyed this Code Pattern? Check out our other [Data Analytics Code Patterns](https://developer.ibm.com/code/technologies/data-science/)
+* **AI and Data Code Pattern Playlist**: Bookmark our [playlist](https://www.youtube.com/playlist?list=PLzUbsvIyrNfknNewObx5N7uGZ5FKH0Fde) with all of our Code Pattern videos
+* **With Watson**: Want to take your Watson app to the next level? Looking to utilize Watson Brand assets? [Join the With Watson program](https://www.ibm.com/watson/with-watson/) to leverage exclusive brand, marketing, and tech resources to amplify and accelerate your Watson embedded commercial solution.
+* **Data Science Experience**: Master the art of data science with IBM's [Data Science Experience](https://datascience.ibm.com/)
+* **PowerAI**: Get started or get scaling, faster, with a software distribution for machine learning running on the Enterprise Platform for AI: [IBM Power Systems](https://www.ibm.com/ms-en/marketplace/deep-learning-platform)
```

Comment:
you should remove the ones that aren't related.  No powerai, spark, k8s.  And above I'd remove Data Analytics Code Patterns.